tests/machine/acorn

```diff
@@ -1,5 +1,5 @@
   module load ecflow/5.6.0.13
```

Comment:
I am assuming you applied these changes to the `machine` subdirectory files manually? Thanks for doing that, let me know in the future if you want help resolving testing conflicts until we can propose these changes back to the ufs-community repository

Reply:
For the NCAR files, ya, I think I did that. It was a while ago now.